### PR TITLE
Adds frontend for ticket

### PR DESCRIPTION
**Pull request by SaptakS**
Frontend for ticket section. All the details are in the yaml file and hence can be edited without having to adjust the frontend.

### CODE REVIEW

File: _data/tickets.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+- name: Early Bird Ticket
+  brief: Sale ends by 30 Sep, 2018 11:59 PM (Asia/Kolkata) OR completion of 300 tickets
+  cost: 1200
+  type: conference
+  status: few
+  url: https://example.com
```
Comment (SaptakS) [inline _data/tickets.yml]: Update the ticket URLs for the specific sections here cc @sayanchowdhury

**Review by palnabarun — APPROVED** [review]
This looks great! Thank you @SaptakS :tada:

**Comment by Sc0rpi0n101** [comment]
Looks Great 🔥 🔥 

We should add the ticket section to the navbar too.

File: _data/tickets.yml (hunk 1 above)
Comment (inovizz) [reply _data/tickets.yml]: Hi @SaptakS, Could you please make the changes for the tickets page based on the following information - 

Early bird  - Rs.149 
Regular ticket - Rs.199 
Late bird tickets - Rs. 499 
Contributor - Rs 1499 min

There is no student ticket this time and here is the townscript event [page](https://www.townscript.com/e/pycon-india-2020) which you can use for URL attribute.

Comment (inovizz) [reply _data/tickets.yml]: And here are the dates for each category of tickets

Early Bird - 17th Aug 12:00 am to 22 Aug 9:00 am
Regular Bird - 22nd Aug 9:00 am to 15th Sep 9:00 am
Late Bird - 15th Sep 9:00 am - 1st Oct 12:00 am
Contributor - 17th Aug 12:00 am to 1st Oct 12:00 am

cc @sayanchowdhury

**Pull request merged**